### PR TITLE
dnx-watch -> dotnet-watch

### PR DESCRIPTION
**Pull request by victorhurdugaci**
- Removed support for dnx
- Added support for dotnet
- Command line parsing is still flaky because of a bug in dotnet. You must run the watcher from the same folder as the project being watched
- Tests not working because there's not xunit runner

Part of https://github.com/aspnet/dnx-watch/issues/34

Please review @davidfowl @anurse

### CODE REVIEW

File: dotnet-watch.sln [hunk 1]
```diff
@@ -17,7 +17,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{442A6A17-4C5A-4E11-B547-A554063FD338}"
 EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Dnx.Watcher.Tests", "test\Microsoft.Dnx.Watcher.Tests\Microsoft.Dnx.Watcher.Tests.xproj", "{640D190B-26DB-4DDE-88EE-55814C86C43E}"
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.DotNet.Watcher.Tests", "test\Microsoft.DotNet.Watcher.Tests\Microsoft.DotNet.Watcher.Tests.xproj", "{640D190B-26DB-4DDE-88EE-55814C86C43E}"
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.DotNet.ProjectModel", "..\..\DotNet\cli\src\Microsoft.DotNet.ProjectModel\Microsoft.DotNet.ProjectModel.xproj", "{303677D5-7312-4C3F-BAEE-BEB1A9BD9FE6}"
```
Comment (davidfowl) [inline dotnet-watch.sln]: Uh no :smile: Use the package

Comment (victorhurdugaci) [inline dotnet-watch.sln]: Whoops, leftover

File: src/Microsoft.DotNet.Watcher/Program.cs [hunk 2]
```diff
@@ -38,38 +37,38 @@ public static int Main(string[] args)
...
-        private int MainInternal(string[] watchArgs, string[] dnxArgs, CancellationToken cancellationToken)
+        private int MainInternal(string[] watchArgs, string[] dotnetArgs, CancellationToken cancellationToken)
         {
             var app = new CommandLineApplication();
-            app.Name = "dnx-watch";
-            app.FullName = "Microsoft .NET DNX File Watcher";
-
+            app.Name = "dotnet-watch";
+            app.FullName = "Microsoft .NET File Watcher";
+
```
Comment (anurse) [inline src/Microsoft.DotNet.Watcher/Program.cs]: Looks like you could use this: https://visualstudiogallery.msdn.microsoft.com/a204e29b-1778-4dae-affd-209bea658a59

**Comment by anurse** [comment]
Looks good as far as I can tell. I wasn't terribly familiar with the previous dnx-watch architecture but the renames all look right to me.

**Comment by muratg** [comment]
Looks good, :shipit:

**Comment by victorhurdugaci** [comment]
Will send an updated PR soon

**Pull request closed without merging**